modules/local/perl/bioperl/main.nf

```diff
@@ -0,0 +1,30 @@
+process PERL_BIOPERL {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::perl-bioperl=1.7.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/perl-bioperl:1.7.2--pl526_11' :
+        'quay.io/biocontainers/perl-bioperl:1.7.2--pl526_11' }"
+
+    input:
+    tuple val(meta), path(file)
+
+    output:
+    tuple val(meta), path("${task.ext.prefix}"), emit: out
+    path "versions.yml"                        , emit: versions
```

Comment:
Change output parameter `out` to the output file type (e.g. `bam` , `tsv`)

Reply:
I updated into `txt`. 
The idea is to keep it very generic, the point of this module is to run any Perl one-liner on any text file.

Reply:
Ah I see, then it makes sense how you had it before. Your choice!